www/pages/settingup_background_map.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+permalink: /howto/settingup_background_map
+---
+
+When surveying in the field, it is essential to have appropriate background maps. There are several sources of online and offline background maps you can use in your QGIS project. Note that you need to comply with the terms of use of the background maps and thier data sources. This page simply explains how to add the data to your QGIS maps and use it in Input. It will be the sole responsibility of the end user to comply with such terms and conditions.
+
+# Raster tiles
+
+## Online services
+
+[QGIS comes by default with the OpenStreetMap online services for XYZ tiles](https://docs.qgis.org/3.10/en/docs/user_manual/managing_data_source/opening_data.html?highlight=xyz#using-xyz-tile-services). When adding cartographic basemap, ensure you set the tile size correctly, so that the texts and labels are readable on mobile devices with high resolution display.
+
+You can also add [other sources](https://gis.stackexchange.com/questions/20191/adding-basemaps-from-google-or-bing-in-qgis/217670#217670) of XYZ tiles to your QGIS.
+
+## Generating raster tiles
+QGIS also offers a [processing algorithm](https://docs.qgis.org/3.10/en/docs/user_manual/processing_algs/qgis/rastertools.html) to generate your own XYZ tiles for offline use.
```

Comment:
Maybe worth linking to a separate article that explains how to use the algorithm step by step for some sample area...